apps/docs/src/components/PageTemplate.astro

```diff
@@ -1,16 +1,36 @@
 ---
 import { parseComponent } from "@selleo/core";
 import { Code } from "astro/components";
-const { component: Component, name, ...props } = Astro.props;
+import { MobileViewIFrame } from "./MobileViewIFrame";
+const {
+  component: Component,
+  name,
+  mobileViewPath,
+  templateHeight,
+  ...props
+} = Astro.props;
 const { HTMLString, ReactString } = parseComponent(Component, props);
 ---
 
 <div class="preview">
   <h3 class="mb-2 font-bold leading-none text-lg">Preview - {name}</h3>
 
-  <div class="relative bg-white h-[772px] mb-2">
-    <Component {...props} />
-  </div>
+  {
+    mobileViewPath ? (
+      <MobileViewIFrame
+        client:load
+        templateHeight={templateHeight}
+        mobileViewPath={mobileViewPath}
+      />
+    ) : (
+      <div
+        class="relative bg-neutral-200 dark:bg-neutral-800 mb-2"
+        {...(templateHeight ? { style: { height: templateHeight } } : {})}
```

Comment:
What do you think about adding default height here?

Reply:
that way it also would shortent to style={{ height: templateHeight || defaultHeight }}

Reply:
Good point! 👍